.env.example.dev

```diff
@@ -30,3 +30,6 @@ CRAFT_SECURITY_KEY=
 CRAFT_DEV_MODE=true
 CRAFT_ALLOW_ADMIN_CHANGES=true
 CRAFT_DISALLOW_ROBOTS=true
+
+# Asset Settings
+ASSET_FILESYSTEM=local
```

Comment:
In the future, the env var would let you choose s3 or local depending on your environment or preference.

Future ticket for some default AWS setup
- https://github.com/vigetlabs/craft-site-starter/issues/15